amd/build/mod_hsuforum_loader.min.js

```diff
@@ -1 +1 @@
\ No newline at end of file
+define(["jquery"],function(e){return registerPostsObserver=function(){const n=e(".hsuforum-post-wrapper"),t=e(".hsuforum-footer-reply")[0];null!=t&&n.push(t),n&&e(n).each(function(){new MutationObserver(()=>{let n=e(this).find("form");if(n){let t=e(n).find(".hsuforum-textarea");e(n).on("submit",()=>{0!=e(t).text()&&document.body.dispatchEvent(spinnerStartEvent)})}}).observe(this,{subtree:!0,childList:!0})})},e("select.postratingmenu.ratinginput").change(function(){e(this).closest(".hsuforum-post-wrapper").find("span.hsuforum-unreadcount").hide()}),startSpinnerHandler=function(){e("#hsuforum-overlay-box").show(),e("#hsuforum-loading-container").show()},stopSpinnerHandler=function(){e("#hsuforum-overlay-box").hide(),e("#hsuforum-loading-container").hide()},registerSpinnerEvents=function(){const e=new Event("spinnerStartEvent"),n=new Event("spinnerStopEvent");window.spinnerStartEvent=e,window.spinnerStopEvent=n,document.body.addEventListener("spinnerStartEvent",()=>{startSpinnerHandler()}),document.body.addEventListener("spinnerStopEvent",()=>{stopSpinnerHandler()})},{init:function(){(function(e,n=0){const t=(new Date).getTime();return new Promise((o,r)=>{const s=setInterval(()=>{const i=(new Date).getTime();document.querySelector(e)?(clearInterval(s),o()):n&&i-t>=n&&(clearInterval(s),r())},100)})})("body",3e4).then(function(){e(".container :input").prop("disabled",!1),e(".mod-hsuforum-posts-container").show(),e("#hsuforum-loading-container").hide(),e("#hsuforum-overlay-box").hide(),window.hascompleted=!1;var n=window.location.hash;window.hascompleted||(n&&(e(n).closest("li").data("depth")>0&&e(n).closest(".posts-collapse-container").addClass("show").find(".posts-collapse-toggle.collapse-bottom").attr("aria-expanded","true"),e("html,body").animate({scrollTop:e(n).offset().top-60},1e3)),window.hascompleted=!0)}).catch(()=>{throw e(".container :input").prop("disabled",!1),e(".article").show(),e("#hsuforum-loading-container").hide(),e("#hsuforum-overlay-box").hide(),"element did not load in 30 seconds"}),registerPostsObserver(),registerSpinnerEvents()}}});
```

Comment:
Has this file been minified with Grunt? If not @BrendonPretorius can you assist since I know @loyisongq grunt is not working

Reply:
If this is an issue please then remove es6 syntax